statsmodels/stats/tests/test_proportion.py

```diff
@@ -995,3 +995,116 @@ def test_ci_symmetry_array(count, method):
     a = proportion_confint([count, count], n, method=method)
     b = proportion_confint([n - count, n - count], n, method=method)
     assert_allclose(np.array(a), 1.0 - np.array(b[::-1]))
+
+def test_proportion_confint_default():
+
+    # Test Default side
+    ci_low, ci_upp = proportion_confint(1, 1)
+    assert_equal(ci_low, 1.0)
+    assert_equal(ci_upp, 1.0)
+
+    ci_low, ci_upp = proportion_confint(100, 200)
+    assert_almost_equal(ci_low, 0.4307048087825161)
```

Comment:
where do the comparison numbers come from?

we use now mainly assert_allclose with atol and/or rtol
(usage of assert_almost_equal comes mostly from before numpy had assert_allclose)

Reply:
These numbers come from the latest version of statsmodels.stats.proportion_confint. Because I did not alter the calculation logic, those numbers must not be changed.
Should I change all assert_almost_equal to assert_allcose?

Reply:
usually we add comparison numbers from other packages, mainly from R for stats.
A `#` comment should mention the source, `regression tests` if numbers are from an earlier statsmodels version.